bleak/backends/bluezdbus/client.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, address, **kwargs):
 
         self._disconnected_callback = None
 
-        self._char_path_to_uuid = {}
+        self._char_path_to_handle = {}
```

Comment:
A comment explaining that this maps D-Bus paths to a BLE handles would be helpful.

Reply:
Good idea.